topics/admin/tutorials/data-library/slides.html

```diff
@@ -0,0 +1,54 @@
+---
+layout: base_slides
+logo: "GTN"
+
+title: "Storage Management"
+questions:
+- How does Galaxy locate data?
+- How can I have Galaxy use multiple storage locations?
+objectives:
+- Setup Galaxy with both the Hierarachical and Distributed Object Storages
+key_points:
+- The distributed object store configuration allows you to easily expand that storage that is attached to your Galaxy.
+- You can move data around without affecting users.
+contributors:
+  - natefoo
+  - martenson
+  - hexylena
+---
+
+
+# Data Libraries
+
+Provide a way to conveniently share Galaxy datasets within a group of Galaxy users or with everybody that has access to a specific instance of Galaxy.
+
+* Can import data from filesystem *without duplicating* it.
+
+--
```

Comment:
I'm not a big fun of incremental slides (unless there is a good reason), but maybe that's just me.

Reply:
nah same. just copied/pasted. I need to seriously reduce amount of text per slide too.